### DISCUSSION TITLE
Using dlj for trading ?

### DISCUSSION THREAD

**Discussion by straumat** [deepjavalibrary/djl#963] [Q&A] [answered]
I'm the developer of [Cassandre](https://github.com/cassandre-tech/cassandre-trading-bot), an open-source trading bot framework available as a Spring boot starter.

My framework allows classical trading, technical analysis trading and I'm now thinking of creating an archetype for trading with machine learning. Do you think your library is suited for that? Do you have examples I could reuse to test it?

**Comment by zachgk** [comment, marked as answer]
Yeah, DJL should be a great solution if you are interested in machine learning in Java.

If you are looking for a basic proof-of-concept, I recommend looking at our [TrainSeq2Seq](https://github.com/deepjavalibrary/djl/blob/master/examples/src/main/java/ai/djl/examples/training/TrainSeq2Seq.java) example. It is used for an NLP machine translation dataset in the example, but it shares a lot of principles as a 1D seq-to-seq model with time series forecasting for trading.

**Reply by straumat** [reply]
@zachgk thx! I will take a look at it and integrate DJL to Cassandre!
